app/assets/stylesheets/_header.scss

```diff
@@ -0,0 +1,67 @@
+.site-header {
+  height: 60px;
+  background: #263039;
+  color: #a8a8a8;
+  font-size: 17px;
+}
+
+.brand-short {
+  background: #414367;
+  color: #d6d6d6;
+  padding: 15px;
+  margin-right: 10px;
+  font-size: 20px;
+}
+
+.brand-short, .brand-long, .header-right-nav div {
+  display: inline-block;
+}
+
+.brand-long {
+  border-right: 1px solid #535353;
+  padding: 10px;
+}
+
+.settings-icon {
+  border-left: 1px solid #535353;
+}
+
+.settings-icon i {
```

Comment:
Consider using scss nesting for the i rather than explicitly listing .settings-icon 

```
.settings-icon {
  border-left: 1px solid #535353;
  i { 
    // style here
  } 
}
```